opta/commands/deploy.py

```diff
@@ -156,3 +160,17 @@ def deploy(
         image_digest=image_digest,
         detailed_plan=detailed_plan,
     )
+
+
+def __check_layer_and_image(layer: "Layer", image: str) -> str:
+    k8s_module = layer.get_module_by_type("k8s-service")
+    image_name = k8s_module[0].data.get("image")
+    if image_name == "AUTO" and image is None:
+        raise UserErrors(
```

Comment:
Don't print an error-- this will just remove the existing pods running, and is useful when doing the first deploy